pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -90,7 +104,8 @@ func NewHandlerImpl(updateCapacityFunc func(v1.ResourceList)) (*HandlerImpl, err
 	deviceManagerMonitorCallback := func(resourceName string, added, updated, deleted []pluginapi.Device) {
```

Comment:
extract this function into something like `defaultCallback`?

Reply:
Do you mean define this as func (h *HandlerImpl) defaultCallback(...)?

Reply:
would it be better to rename the function to indicate that its common callback for all resources, something like genericDeviceSyncCallback?
Not sure if @tengqm meant the same or similar.

Reply:
This functions seems to be getting invoked only from unit tests. I am unable to understand what purpose this function, `deviceManagerMonitorCallback` has.
Looks to me we are adding a function which is not being used anywhere and then writing unit test to test its logic.

Please suggest if i misunderstood something.